### PR TITLE
[FIX] l10n_de_stock: bad forward port

### PR DESCRIPTION
**Pull request by william-andre**
Following
https://github.com/odoo/odoo/commit/bdcb3d192be1e01e1141aa09c60027337009a67b

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_de_stock/models/stock.py [hunk 1]
```diff
@@ -3,12 +3,16 @@
 
 
 class StockInventory(models.Model):
-    _inherit = 'stock.inventory'
+    _inherit = 'stock.picking'
```
Comment (qdp-odoo) [inline addons/l10n_de_stock/models/stock.py:6]: heu? but the class stays StockInventory ?

Comment (william-andre) [reply addons/l10n_de_stock/models/stock.py:6]: Not anymore

Comment (qdp-odoo) [reply addons/l10n_de_stock/models/stock.py:6]: better

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**